crates/view/src/client.rs

```diff
@@ -763,6 +763,7 @@ where
         async move {
             let address = self2.address_by_index(tonic::Request::new(pb::AddressByIndexRequest {
                 address_index: Some(address_index.into()),
+                display_confirm: false,
```

Comment:
Not particularly important here, but I think it's slightly cleaner to do
```
..Default::default()
```
instead; the result is the same but the syntax signals that we don't care about the `display_confirm` value rather than that we explicitly want it to be false for some reason.